src/Microsoft.AspNetCore.Razor.Runtime/project.json

```diff
@@ -56,9 +56,12 @@
     },
     "netstandard1.3": {
       "dependencies": {
+        "System.Collections.Concurrent": "4.0.12-*",
+        "System.Reflection": "4.1.0-*",
```

Comment:
Hmm, i'm not sure upgrading `System.Reflection.Extensions` to retrieve a new `System.Reflection` when we really don't need a higher extensions package is the right approach; however, if this is the recommended guidelines I'd be more than happy to do it.

/cc @Eilon is this something we'd typically do/recommend?